src/lighteval/metrics/__init__.py

```diff
@@ -95,22 +95,18 @@ def apply_generative_metric(
         preds = [formatted_doc.specific["label_to_choices"].get(p) for p in preds]
         golds = [formatted_doc.specific["label_to_choices"][g] for g in golds]
 
+    preds_no_sampling = preds
+    if max_num_samples > 1:  # We want to run our evaluation on only one sample for base generative evals
+        preds_no_sampling = as_list(preds[0])
```

Comment:
do this comes from this PR ?

Reply:
Could be a merge issue - but the max_num_samples comparision is different though, so I'm unsure